### PR TITLE
Updated README to English, changed the examples and described the multi-module/component setup

### PR DESCRIPTION
**Pull request by mcassiano**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,204 +1,151 @@
...
+
+![](art/artboard.png)
+
+JSON Specification
 -------------------------------
-O json que foi definido para que a lib consiga fazer a conversão a partir de uma estrutura padrão
+
+The JSON specification is very simple. We only have two entity models that describe how to represent a view (component) 
+and how to style these components (properties). Our goal was to make the library as extensible as possible,
+so you can create new components or properties as you grow you project (don't forget to contribute some of them 😊).
```
Comment (mcassiano) [inline README.md]: your* project

File: README.md [hunk 2]
```diff
@@ -1,204 +1,151 @@
...
+val parent = findViewById<FrameLayout>(R.id.parent)
+
+val json = MainActivity::class.java
+    .getResource("/assets/example_simple.json")
+    .readText()
+
+val creator = DynamicViewCreator.Builder()
+    .build()
+
+creator.createViewFromJson(parent, json, this)
```
Comment (cs-yasmin-pereira) [inline README.md]: How about add some comments about the code here? Like "The json that you want to use" or "This is the view creator, you can use the default builder or add your own components".
The code does not mention the action listener, just passing "this" as a parameter

**Review by cs-yasmin-pereira — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (mcassiano) [reply README.md]: I think there should be a section talking about the action listener, but not redundant comments like "this is the json you want to use". There's also a section about custom components so I don't think the other comment brings value either. IMHO.

Comment (cs-yasmin-pereira) [reply README.md]: Sure, i also think that this section should exist. But in this context the README is just passing "this" as a parameter. Maybe the section should be before this code and show that the activity is implementing the listener

Comment (mcassiano) [reply README.md]: Yup, agreed. Will work on that.

**Review by cs-yasmin-pereira — APPROVED** [review]

**Pull request merged**